system/Database/BaseBuilder.php

```diff
@@ -1130,6 +1137,48 @@ protected function _like_statement(?string $prefix, string $column, ?string $not
         return "{$prefix} {$column} {$not} LIKE :{$bind}:";
     }
 
+    /**
+     * Add UNION statement
+     *
+     * @param BaseBuilder|Closure $union
+     *
+     * @return $this
+     */
+    public function union($union)
+    {
+        return $this->addUnionStatement($union);
+    }
+
+    /**
+     * Add UNION ALL statement
+     *
+     * @param BaseBuilder|Closure $union
+     *
+     * @return $this
+     */
+    public function unionAll($union)
+    {
+        return $this->addUnionStatement($union, true);
+    }
+
+    /**
+     * @used-by union()
+     * @used-by unionAll()
+     *
+     * @param BaseBuilder|Closure $union
+     *
+     * @return $this
+     */
+    protected function addUnionStatement($union, bool $all = false)
+    {
+        $this->QBUnion[] = "\n" . 'UNION '
+            . ($all ? 'ALL ' : '')
+            . 'SELECT * FROM '
+            . $this->buildSubquery($union, true, 'uwrp' . (count($this->QBUnion) + 1));
```

Comment:
What is `uwrp`?

Reply:
alias

Reply:
Why is it the name `uwrp`? 
What's the meaning?

Reply:
union wrapper =) 

If you can suggest another name for the alias, I'd be grateful.

Reply:
Aha! 
It is difficult to name it. I'm okay with uwrp.